packages/ui/LogoSettingModal/components/LogoSetting.tsx

```diff
@@ -0,0 +1,244 @@
+        };
+        await updateProductAction(product.id, inputProduct);
+      }
+      setLocalProduct &&
+        setLocalProduct({
+          ...product,
+          brand: { logoUrl: replacedLogo, bgColor: backgroundColor },
+        });
+      setIsImageAddedFromAddLogoButton && setIsImageAddedFromAddLogoButton(true);
+      setImage && setImage(replacedLogo);
+
+      setImageUrlFromLogoButton && setImageUrlFromLogoButton(replacedLogo);
+      toast.success("Logo uploaded successfully.");
+    } catch (error) {
+      if (error instanceof Error) {
+        toast.error(`Error: ${error.message}`);
+      } else {
+        throw error;
+      }
+    }
+  };
+
+  const toggleaddBackgroundColor = (checked: boolean) => {
+    if (!checked) {
+      setBackgroundColor("");
+    } else {
+      setBackgroundColor(product?.brand?.bgColor);
+    }
+
+    setAddBackgroundColor(checked);
+  };
+
+  return (
+    <>
+      <div className="relative">
+        {!isEdit && fromLookAndFeelSetting && <div className="absolute z-30 h-full w-full"></div>}
+        {isLoading && (
+          <div className="absolute inset-0 flex items-center justify-center">
+            <LoadingSpinner />
+          </div>
+        )}
+        <div className="flex flex-col gap-8">
+          <div className="flex flex-col gap-3">
+            <div className="text-sm font-semibold text-slate-700">Preview</div>
+            {!isStandardFileUploadOpen ? (
+              <Image
+                src={`${replacedLogo ? replacedLogo : imageUrl}`}
+                alt="logo"
+                style={{ backgroundColor: backgroundColor }}
+                className="h-20 w-auto max-w-64 rounded-lg border object-contain p-1 "
+                width={256}
+                height={56}
+              />
+            ) : (
+              <FileInput
+                id="Companylogo-input"
+                allowedFileExtensions={["png", "jpeg"]}
+                environmentId={environmentId}
+                onFileUpload={(url: string[] | undefined) => {
+                  if (url && url.length > 0) {
+                    setReplacedLogo(url[0]);
+                    setIsStandardFileUploadOpen(false);
+                  }
+                }}
+              />
+            )}
+
+            <div className="flex gap-4">
+              {(isEdit || !fromLookAndFeelSetting) && (
+                <>
+                  <Button
+                    variant="secondary"
+                    onClick={() => {
+                      if (replaceLogoRef.current) {
+                        replaceLogoRef.current.click();
+                      }
+                    }}
+                    disabled={isLoading}>
+                    Replace logo
+                  </Button>
+                  <Input
+                    ref={replaceLogoRef}
+                    className="hidden"
+                    type="file"
+                    accept="image/jpeg,image/png"
```

Comment:
We'd like to support all image mimes so can you try changing this with `image/*`?